### DISCUSSION TITLE
Question about data resampling

### DISCUSSION THREAD

**Discussion by oliver-zhang987** [open-mmlab/mmclassification#1506] [Q&A]
I didn't see the description of data resampling in the mmpretrain tutorial. Is the data resampled by default? If not, how to set up data category balanced resampling？

**Comment by Ezra-Yu** [comment]
you can use this https://mmengine.readthedocs.io/zh_CN/latest/api/generated/mmengine.dataset.ClassBalancedDataset.html#mmengine.dataset.ClassBalancedDataset
